src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -59,6 +60,7 @@ public static partial class PlatformDetection
         public static bool IsBrowserDomSupported => GetIsBrowserDomSupported();
         public static bool IsBrowserDomSupportedOrNotBrowser => IsNotBrowser || GetIsBrowserDomSupported();
         public static bool IsNotBrowserDomSupported => !IsBrowserDomSupported;
+        public const TestPlatforms LocalEchoServerIsNotAvailable = TestPlatforms.Any & ~TestPlatforms.Browser;
```

Comment:
Maybe this should be named like `RemoteEchoServerAvailable`? Because the local echo server can be available for any project. So, something like: 

```csharp
public static bool RemoteEchoServerAvailable => !IsBrowser;
```

And correspondingly:

```csharp
[OuterLoop("Uses external servers", typeof(PlatformDetection), nameof(PlatformDetection.RemoteEchoServerAvailable)]
```

.. following the pattern used for `PlatformDetection` in the tests.

Reply:
done